apps/ui/src/helpers/ens.ts

```diff
@@ -73,3 +125,10 @@ export async function getNameOwner(name: string, chainId: ENSChainId) {
     }
   );
 }
+
+export async function getSpaceController(name: string, chainId: ENSChainId) {
+  const snapshotRecord = await getEnsTextRecord(name, 'snapshot', chainId);
+  if (snapshotRecord) return snapshotRecord;
```

Comment:
There is another case, where spaces can have a text record like 
```
ipns://storage.snapshot.page/registry/<CONTROLLER_ADDRESS>/aave.eth
```

https://app.ens.domains/aave.eth?tab=records

In this case `0x60C8dC4762b217b4A00FF1824111077f331B1FbF` will be controller. but we resolve this
<img width="204" alt="image" src="https://github.com/user-attachments/assets/c8fbcd41-0482-44e9-9fcf-c1044a53de9c">


Refer to https://github.com/snapshot-labs/snapshot.js/blob/master/src/utils.ts#L628-L645 on how to parse the URI

Reply:
Refer
https://github.com/snapshot-labs/snapshot.js/blob/master/src/utils.ts#L536-L578

There is a chance that a ENS name can change its resolver, so we should get resolver first, to see if we support it. and then read text record